lighthouse/config/defaults.py

```diff
@@ -200,3 +205,10 @@
     "instrument_loaded_incorrectly": "Labware has been incorrectly loaded onto instrument",
     "other": "Any other failure",
 }
+
+BIOSERO_ROBOTS = {
```

Comment:
Not sure I understand this config. What are the BHRB ids?
Can this distinguish between the workcell and the liquid handler robot?
In the cherrytrack DB table we have an 'automation_system_name' for the 2 workcells ('CPA' and 'CPB') and an 'automation_system_manufacturer' ('Biosero').
We also have a 'liquid_handler_serial_number' for the Hamilton (e.g. SN12345678).

Reply:
This does not distinguish, they are just references sent to us from the Biosero system. If they want to refere to individual parts of the workcell we can add them here with different barcodes, but I suppose is going to be the same barcode for the whole workcell.